autoscale-source-dockerswarm/pom.xml

```diff
@@ -71,5 +58,13 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi.util</groupId>
```

Comment:
Why are these 2 dependencies listed below the test dependencies? Test dependencies should be listed last.

Reply:
Fixed